app/api/sessions.py

```diff
@@ -59,8 +60,15 @@ def before_post(self, args, kwargs, data):
         ):
             raise ForbiddenError({'pointer': ''}, "Sessions are disabled for this Event")
 
+        
+        custom_fields = db.session.query(CustomForms).filter_by(event_id = data['event']).all()
+        excluded = ['track']
+        for i in custom_fields:
+            if(i.field_identifier == "track" and i.is_required == True):
+                excluded = []   
```

Comment:
excluded shouldn't be needed now

Reply:
means only exclude = ["track"] is enough?